sympytorch/sympy_module.py

```diff
@@ -66,25 +66,29 @@ def fn_(*args):
 class _Node(torch.nn.Module):
     def __init__(self, *, expr, _memodict, _func_lookup, **kwargs):
         super().__init__(**kwargs)
-
+        
```

Comment:
Can spurious changes be avoided, to keep a clean diff?

Reply:
yep